docs/meshstack.billing-configuration.md

```diff
@@ -510,28 +510,63 @@ let example
 
+#### meshStack Default Exchange Rates
 
-### Multi-Currency
+Fetches rates from [frankfurter.app](https://www.frankfurter.app/), using exchange rates from the European Central Bank.  
 
-If you decide not to use currency conversion and prefer to have multiple currencies in chargeback statements, simply disable the toggle on the Financial Tab in the Settings page.
+**To enable**:
 
-The following product functionalitities do currently not support multi-currency scenarios:
+- Go to the Financials tab and select **meshStack default exchange rates**.  
+- If meshStack is hosted privately, ensure it can make external requests (configure firewall rules if needed). 
+- If external access is restricted and configuration is not done in meshStack, **currency conversion will not apply**.
+
+
+
+#### Custom API Exchange Rates
+
+You can provide exchange rates of your company via API.
+
+
+**To enable**:
+
+- Go to the Financials tab and select **Custom API Exchange Rates**.
+- Contact **support@meshcloud.io** early to set this up before chargeback finalization.  
+- If no rates are provided by the finalization date and the currency converter is turned on, **meshStack default exchange rates** will apply for that month.
+
+  
+
+
+For chargeback, we use the daily exchange rate from the day your statement is finalized, regardless of whether you're using your own converter or the default one. For example, if your chargeback finalization day is March 31st, and the USD to Euro rate on that day is 0.93, then all March chargeback statements in USD will be converted and finalized at this rate.
+
+
+### Key Points: Currency Converter Usage & Limitations
+
+When you choose the currency converter type and **turn it ON** then you will have all your open chargebacks and usage reports converter to euro. if you choose your own API exchange rate 
+
+By default, the currency converter is **off**, meaning all chargebacks and usage reports are in their original cloud provider currency. 
+If you turn on the currency converter and later decide to turn it off, all future chargebacks, as well as past finalized chargebacks, will revert to being displayed in their original currency.
+
+#### Limitation
+
+1. **Future-only impact**: Currency conversion only applies to data after activation. Past finalized chargebacks and usage reports remain in their original currency.
+2. **Euro-only conversion**: Currently, conversion supports **only to Euro**but we plan to support other currencies in the future.
+3. **Payment Methods**: 
+    Payment method amounts are currently limited to EUR, but we plan to support additional currencies in the future.
+4. **Prices for Platforms and Building Blocks**: Currently, prices can only be set in EUR, but we plan to support other currencies in the future.
+5. **Known Issue with Timing**: If a tenant has costs in a month **before activation** and no further costs are recorded afterward, those costs will remain in the original currency.  
```

Comment:
f: this sentence is also not correct:

> 5. **Known Issue with Timing**: If a tenant has costs in a month **before activation** and no further costs are recorded afterward, those costs will remain in the original currency.

Because of the reasons mentioned earlier: in the background, currency conversion is always applied. The setting just changes if we expose converted amounts to the customer or not.